src/lib/import-export/import/importers/importer.ts

```diff
@@ -156,6 +157,30 @@ abstract class BaseBackupImporter extends BaseImporter {
 		await shell.trashItem( dbPath );
 	}
 
+	protected async moveExistingWpContentToTrash( rootPath: string ): Promise< void > {
+		const wpContentDir = path.join( rootPath, 'wp-content' );
+		try {
+			if ( ! fs.existsSync( wpContentDir ) ) {
+				return;
+			}
+			const contentToKeep = [ 'mu-plugins', 'database', 'db.php' ];
+
+			const contents = await fsPromises.readdir( wpContentDir );
+
+			for ( const content of contents ) {
+				const contentPath = path.join( wpContentDir, content );
+
+				if ( contentToKeep.includes( content ) ) {
+					continue;
+				}
+
+				await shell.trashItem( contentPath );
```

Comment:
Let's delete those instead of trashing them. Running multiple imports produces many plugins/themes/uploads entries in the trash, which can be confusing. The confirmation window already makes it clear that import will replace all files and the database.

Ideally, we back up the whole site directory before import starts and revert to that state if import fails, but it would be an idea for another issue. The current solution tries to implement that, but not in a robust way.